transcom/mymove
# MAIN - B-20131 sc edit full address

## [B-20131](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20131)

INT PRs: 
https://github.com/transcom/mymove/pull/13013
https://github.com/transcom/mymove/pull/13120

## Summary

Added an edit icon button to `Starting Address` and `Ending Address` on Review Documents page when reviewing as a closeout SC. The edit button opens up a Modal where you can edit values of these.

### How to test

1. Use `MoveWithPPMShipmentReadyForFinalCloseout` testharness to create a move.
2. As a customer -> go to the move and complete uploading PPM documents.
3. As a closeout SC -> find the move and click "Review Documents".
4. Click "Show details" arrow button next to Shipment Info.
5. You should see a pencil icon next to "Starting Address" -> Click on it.
6. Change the value in the modal and save.
7. Should see the change reflected.
8. Repeat steps 5-7 for "Ending Address".

## Screenshots
![image](https://github.com/transcom/mymove/assets/146856854/ac2ba8e2-f9ce-4126-a0b2-6196cc6a4995)
